internal/api/registry/blobs_test.go

```diff
@@ -382,7 +381,7 @@ func TestBlobStreamedAndChunkedUpload(t *testing.T) {
 					Body:         assert.ByteData(blob.Contents),
 					ExpectStatus: http.StatusAccepted,
 				}.Check(t, h)
-				uploadURL := resp.Header.Get("Location")
+				uploadURL := resp.Header.Get("Location") //nolint:govet
```

Comment:
Just out of curiosity, what was govet complaining about here?

Reply:
Variable shadowing of `uploadURL`.

Reply:
I see, thanks!